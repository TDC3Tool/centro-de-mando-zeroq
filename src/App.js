import React from 'react'
import { GlobalStyle } from './styles/GlobalStyles'
import { Header } from './components/Header'
import { Content } from './components/Content'

import 'bulma/css/bulma.css'

export const App = () => (
  <>
    <GlobalStyle />
    <Header />
    <Content />
  </>
)
