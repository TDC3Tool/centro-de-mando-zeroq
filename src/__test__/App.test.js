import React from 'react'
import { mount } from 'enzyme'
import { create } from 'react-test-renderer'
import { App } from '../App'

describe('<App /> component', () => {
  const app = mount(<App />)

  it('Render', () => {
    expect(app.length).toEqual(1)
  })
})

describe('<App /> spnapshot', () => {
  it('Check the UI of <App />', () => {
    const app = create(<App />)
    expect(app.toJSON()).toMatchSnapshot()
  })
})
