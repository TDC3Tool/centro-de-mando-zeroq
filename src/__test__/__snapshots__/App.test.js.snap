// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> spnapshot Check the UI of <App /> 1`] = `
Array [
  <header
    className="sc-AykKC gcJED"
  >
    <img
      alt="Control"
      src="\\"logo.png\\""
    />
  </header>,
  <section
    className="sc-AykKD jOxhDx"
  >
    <div
      className="container px-2"
    >
      <div
        className="columns is-centered is-mobile"
      >
        <div
          className="column is-10"
        >
          <div
            className="columns is-mobile is-tablet"
          >
            <div
              className="column is-12-mobile is-12-desktop is-5-widescreen"
            >
              <div
                className="field"
              >
                <div
                  className="control has-icons-left"
                >
                  <input
                    className="input"
                    onChange={[Function]}
                    placeholder="Buscar sucursal"
                    type="text"
                    value=""
                  />
                  <span
                    className="icon is-small is-left"
                  >
                    <svg
                      fill="currentColor"
                      height="1em"
                      stroke="currentColor"
                      strokeWidth="0"
                      style={
                        Object {
                          "color": undefined,
                        }
                      }
                      viewBox="0 0 512 512"
                      width="1em"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M505 442.7L405.3 343c-4.5-4.5-10.6-7-17-7H372c27.6-35.3 44-79.7 44-128C416 93.1 322.9 0 208 0S0 93.1 0 208s93.1 208 208 208c48.3 0 92.7-16.4 128-44v16.3c0 6.4 2.5 12.5 7 17l99.7 99.7c9.4 9.4 24.6 9.4 33.9 0l28.3-28.3c9.4-9.4 9.4-24.6.1-34zM208 336c-70.7 0-128-57.2-128-128 0-70.7 57.2-128 128-128 70.7 0 128 57.2 128 128 0 70.7-57.2 128-128 128z"
                      />
                    </svg>
                  </span>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </section>,
  <section
    className="sc-AykKE jmgFSq"
  >
    <div
      className="container"
    >
      <div
        className="columns is-centered is-mobile"
      >
        <div
          className="column is-10 is-12-mobile"
        >
          <div
            className="columns is-multiline is-mobile is-desktop is-tablet"
          />
        </div>
      </div>
    </div>
  </section>,
]
`;
